ui/lib/components/HighlightSQL/index.tsx

```diff
@@ -1,6 +1,9 @@
 import React, { useMemo } from 'react'
 
-import { Light as SyntaxHighlighter } from 'react-syntax-highlighter'
+// This usage will generate tons of files about languages highlight when esbuild splitting enable
+// See https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/master/src/async-languages/hljs.js to understand why
+// import { Light as SyntaxHighlighter } from 'react-syntax-highlighter'
+import SyntaxHighlighter from 'react-syntax-highlighter/dist/esm/light'
```

Comment:
To prevent from generating more than one thousand js files that are related to kinds of language syntax highlight.